dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -73,4 +73,7 @@ END;
 d//
 delimiter ;
 CALL modify_data_value_t_ds_dq_rule_input_entry;
-DROP PROCEDURE modify_data_value_t_ds_dq_rule_input_entry;
\ No newline at end of file
+DROP PROCEDURE modify_data_value_t_ds_dq_rule_input_entry;
+
+DROP TABLE IF EXISTS `t_ds_resources`;
+DROP TABLE IF EXISTS `t_ds_relation_resources_user`;
```

Comment:
This is a dangerous and irreversible operation, I don't think it should be added in ddl.

Reply:
Ok, I revert this change, and add `@Deprecated` in `t_ds_relation_resources_user` and `t_ds_resources`.